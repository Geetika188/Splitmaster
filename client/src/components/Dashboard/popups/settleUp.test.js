import React from 'react';
import { mount, configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import SettleUp from './settleUp';

configure({ adapter: new Adapter() });

it('should render correctly', () => {
    let values =["mia","jia","tia"];
  const component = mount(<SettleUp eve={values}/>);
  expect(component).toMatchSnapshot();
  component.unmount();
});
