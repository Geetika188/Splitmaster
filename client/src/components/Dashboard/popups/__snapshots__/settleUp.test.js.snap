// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<SettleUp
  eve={
    Array [
      "mia",
      "jia",
      "tia",
    ]
  }
>
  <div
    className="friendPopup"
  >
    <div
      className="flx"
    >
      <div
        className="frnd-content"
      >
        <div
          className="frnd-header"
        >
          <span>
            Settle up
          </span>
          <button
            className="float-right"
          >
            <i
              class="fas fa-times"
            />
          </button>
        </div>
        <div
          className="frnd-set"
        >
          <button
            onClick={[Function]}
          >
            you
          </button>
           paid 
          <button
            onClick={[Function]}
          >
            select
          </button>
        </div>
        <input
          className="money"
          id=""
          name=""
          onChange={[Function]}
          placeholder="$ 0.0"
          type="number"
        />
        <div
          className="pop-btn bt-mr"
        >
          <button
            className="btn Add"
            onClick={[Function]}
          >
            Save
          </button>
          <button
            className="btn cut"
          >
            Close
          </button>
        </div>
      </div>
      <div
        className="secondBox"
      >
        <div
          className="frnd-header"
        >
          <span>
            Paid To
          </span>
        </div>
        <ul
          className="myList"
        >
          <li
            id="mia"
            onClick={[Function]}
          >
            mia
          </li>
          <li
            id="jia"
            onClick={[Function]}
          >
            jia
          </li>
          <li
            id="tia"
            onClick={[Function]}
          >
            tia
          </li>
        </ul>
      </div>
      <div
        className="secondBox"
      >
        <div
          className="frnd-header"
        >
          <span>
            Paid To
          </span>
        </div>
        <ul
          className="myList"
        >
          <li
            id="mia"
            onClick={[Function]}
          >
            mia
          </li>
          <li
            id="jia"
            onClick={[Function]}
          >
            jia
          </li>
          <li
            id="tia"
            onClick={[Function]}
          >
            tia
          </li>
        </ul>
      </div>
      <div
        className="secondBox"
      >
        <div
          className="frnd-header"
        >
          <span>
            Paid To
          </span>
        </div>
        <ul
          className="myList"
        >
          <li
            id="mia"
            onClick={[Function]}
          >
            mia
          </li>
          <li
            id="jia"
            onClick={[Function]}
          >
            jia
          </li>
          <li
            id="tia"
            onClick={[Function]}
          >
            tia
          </li>
        </ul>
      </div>
    </div>
  </div>
</SettleUp>
`;
